ui/app/AppLayouts/Onboarding2/pages/SyncProgressPage.qml

```diff
@@ -13,19 +13,11 @@ OnboardingPage {
     id: root
 
     required property int syncState // Onboarding.SyncState.xxx
-    property int timeoutInterval: 30000
```

Comment:
Why removed?

Reply:
Same below

Reply:
They weren't needed. We use the state coming in from the View instead. The Timer was just changing the state incorrectly (it was removed by Michal too)

Reply:
Yup, seen it there as well